freqtrade/rpc/api_server/api_v1.py

```diff
@@ -508,3 +509,19 @@ def sysinfo():
 @router.get("/health", response_model=Health, tags=["info"])
 def health(rpc: RPC = Depends(get_rpc)):
     return rpc.health()
+
+
+@router.get("/trades/open/custom-data", response_model=list[ListCustomData], tags=["info"])
+def list_open_trades_custom_data(rpc: RPC = Depends(get_rpc)):
+    """
+    Fetch custom data for all open trades.
+    """
+    return rpc._rpc_list_custom_data()
```

Comment:
I doubt this will scale - if you've got 100ds or 1000nds of trades here - you'll get all entries back, resulting in a call that either takes ages, consumes tons of memory - or (more likely) - both.

As a consequence, this endpoint will probably need some kind of pagination  / limit - probably / most likely similar to the `/trades` endpoint.

Alternatively, a variant that accepts a trade list might also work - with the assumption that you may not want all trades - but just a couple.

Reply:
I've added the pagination management, we will see if the variant accepting a list of trades could be useful.